utilities/common.py

```diff
@@ -130,6 +133,10 @@ def common_parser(for_reco_highPU=False):
         parser.add_argument("--muonCorrEtaBins", default=1, type=int, help="Number of eta bins for dummy muon momentum calibration uncertainty")
         parser.add_argument("--biasCalibration", type=str, default=None, choices=["binned","parameterized", "A", "M"], help="Adjust central value by calibration bias hist for simulation")
         parser.add_argument("--smearing", action='store_true', help="Smear pT such that resolution matches data") #TODO change to --no-smearing once smearing is final
+        parser.add_argument("--validateByMassWeights", 
```

Comment:
I don't think this should go in common since it only works for W (and won't work for low PU)

Reply:
could this also work for the wlike analysis?

Reply:
you are right, it hasn't been implemented at the moment. I put it under common in anticipation that it might be needed in the future for wlike.

Reply:
cool, I added `raise NotImplementedError()` in wlike.